cvat/apps/lambda_manager/views.py

```diff
@@ -445,21 +451,36 @@ def enqueue(self,
         # with invocation of non-trivial functions. For example, it cannot run
         # staticmethod, it cannot run a callable class. Thus I provide an object
         # which has __call__ function.
-        rq_job = queue.create_job(LambdaJob(None),
-            meta = { "lambda": True },
-            kwargs = {
-                "function": lambda_func,
-                "threshold": threshold,
-                "task": task,
-                "job": job,
-                "quality": quality,
-                "cleanup": cleanup,
-                "conv_mask_to_poly": conv_mask_to_poly,
-                "mapping": mapping,
-                "max_distance": max_distance
-            })
+        user_id = request.user.id
+        cm = queue.connection.lock(f'{queue.name}-lock-{user_id}') if settings.LIMIT_ONE_USER_TO_ONE_AUTO_ANNOTATION_TASK_AT_A_TIME else nullcontext()
```

Comment:
Why don't lock always? Why do you need the condition here?

Reply:
I thought it would be better not to use locking in cases when it is not required.